app/controllers/dogs_controller.rb

```diff
@@ -101,6 +92,24 @@ def index
     end
   end
 
+  def manager_index
```

Comment:
I forget the exact permissions here but this route should be protected somehow. `admin?` is there a dog manager role?  I really gotta get on those perms

Reply:
ha, I asked you first!

It looks like we previously just looked for `signed_in?` so I think we're good